<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="2" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="45176" started-at="2020-09-19T14:56:05Z" finished-at="2020-09-19T14:56:50Z">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.test_runner.TestRunner@525d79f0]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Test" duration-ms="45176" started-at="2020-09-19T14:56:05Z" finished-at="2020-09-19T14:56:50Z">
      <class name="com.test_runner.TestRunner">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.test_runner.TestRunner@525d79f0]" name="setUpClass" is-config="true" duration-ms="931" started-at="2020-09-19T14:56:05Z" finished-at="2020-09-19T14:56:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="PASS" signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.test_runner.TestRunner@525d79f0]" name="runScenario" duration-ms="22129" started-at="2020-09-19T14:56:06Z" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2020-09-19T14:56:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA["To Test the Admin Functionality of OrangeHrm"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["To Test the OrangeHrm Application"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method status="PASS" signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.test_runner.TestRunner@525d79f0]" name="runScenario" duration-ms="20486" started-at="2020-09-19T14:56:29Z" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2020-09-19T14:56:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA["To Test the Admin Functionality of OrangeHrm"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["To Test the OrangeHrm Application"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method status="FAIL" signature="Extentsreport()[pri:0, instance:com.test_runner.TestRunner@525d79f0]" name="Extentsreport" is-config="true" duration-ms="147" started-at="2020-09-19T14:56:49Z" finished-at="2020-09-19T14:56:49Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.vimalselvam.cucumber.listener.Reporter.loadXMLConfig(Reporter.java:57)
at com.test_runner.TestRunner.Extentsreport(TestRunner.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:214)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Extentsreport -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:com.test_runner.TestRunner@525d79f0]" name="tearDownClass" is-config="true" duration-ms="1117" started-at="2020-09-19T14:56:49Z" finished-at="2020-09-19T14:56:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.test_runner.TestRunner -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
